### PR TITLE
Nginx setup

### PR DESCRIPTION
**Pull request by ucfchandra**
Nginx setup now working, although still requires to access it at port 8000 as there are permissions issues with leaving it at port 80.

### CODE REVIEW

File: Containerfile [hunk 1]
```diff
@@ -20,19 +20,34 @@ RUN apt-get clean && \
...
-RUN python3 manage.py migrate  
+RUN python3 manage.py collectstatic --noinput  
+RUN python3 manage.py migrate
 
-CMD ["sh", "-c", "gunicorn -c config/gunicorn/dev.py"]
+COPY /config/announcements.conf /etc/nginx/sites-available/
+
+RUN ln -s /etc/nginx/sites-available/announcements.conf /etc/nginx/sites-enabled/announcements.conf
+
+WORKDIR /app/
```
Comment (TimNeubrander) [inline Containerfile:46]: Is redundant Workdir needed?

File: config/gunicorn/dev.py [hunk 2]
```diff
@@ -7,7 +7,7 @@
 # The number of worker processes for handling requests
 workers = 2
 # The socket to bind
-bind = "0.0.0.0:8000"
+bind = "0.0.0.0:127"
```
Comment (jmbarne3) [inline config/gunicorn/dev.py:10]: Is this a typo, or are we wanting to bind gunicorn to port 127?

**Review by jmbarne3 — APPROVED** [review]
We might consider moving the `announcements.conf` file to `config/nginx` to stay consistent with the pattern you started with the `config/gunicorn` directory.

File: config/gunicorn/dev.py (hunk 2 above)
Comment (ucfchandra) [reply config/gunicorn/dev.py:10]: This is what I thought you meant when you said it should be on 127 for security reasons. Unless you meant change 0.0.0.0 to 127.0.0.1?

**Review by Ramin-Farhadi — APPROVED** [review]

File: config/gunicorn/dev.py (hunk 2 above)
Comment (jmbarne3) [reply config/gunicorn/dev.py:10]: Yes, I meant it should be listening specifically on the loopback address - 127.0.0.1

**Pull request merged**